### DISCUSSION TITLE
P10K with warp

### DISCUSSION THREAD

**Discussion by muhannad-hash** [romkatv/powerlevel10k#2592] [General]
I saw you were trying to help warp to implement the p10k.
I just started  to use warp and I really like the p10k the only issue I have is when I use kubectl  commands It doesn't add the current context while I am typing, is this issue gonna be fixed any time soon.

**Comment by romkatv** [comment]
Warp doesn't support a few things that all others terminals have. This is one of them. I don't know whether Warp developers have plans for implementing this.
